src/views/GovernanceOverview/index.test.tsx

```diff
@@ -57,6 +61,7 @@ describe('The governance overview page', () => {
       )
       .toJSON();
 
+    await act(async () => {});
```

Comment:
Can you leave an in-line comment explaining this? Just coming in with no context it's a bit hard to understand why it might be necessary.

Reply:
From looking at the [API docs](https://reactjs.org/docs/test-renderer.html#testrendereract), it looks like the `act()` call should wrap the call to `renderer.create()`? I'm also not sure what's going on here.

Reply:
I had followed the documentation regarding wrapping the act()/await act() around renderer.create() but was still getting a null snapshot when the expect executed.  The renderer.create needed to be resolved because of state changing within the render. State was now changing due to a valid user returned from oktaAuth.  The empty await act() promise was a proposed solution.  I can take another look at this tomorrow.